From: Christoph Junghans <junghans@votca.org> 
Date: 2016-09-09

Newer versions of netcdf's ncdump prints stuff in a diffent order

diff -Naur exodus-6.02.orig/exodus/cbind/test/test1.dmp exodus-6.02/exodus/cbind/test/test1.dmp
--- exodus-6.02.orig/exodus/cbind/test/test1.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/test1.dmp	2016-09-09 10:59:34.378167558 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (10 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (10 currently)
 	num_dim = 3 ;
 	num_nodes = 28 ;
 	num_elem = 8 ;
@@ -174,9 +174,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is testwt1" ;
-		:maximum_name_length = 32 ;
 data:
 
  time_whole = 0.01, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/test2-1.dmp exodus-6.02/exodus/cbind/test/test2-1.dmp
--- exodus-6.02.orig/exodus/cbind/test/test2-1.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/test2-1.dmp	2016-09-09 10:59:34.378167558 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (10 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (10 currently)
 	num_dim = 3 ;
 	num_nodes = 26 ;
 	num_elem = 5 ;
@@ -137,9 +137,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  time_whole = 0.01, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/test2-2.dmp exodus-6.02/exodus/cbind/test/test2-2.dmp
--- exodus-6.02.orig/exodus/cbind/test/test2-2.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/test2-2.dmp	2016-09-09 10:59:34.379167514 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (10 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (10 currently)
 	num_dim = 3 ;
 	num_nodes = 26 ;
 	num_elem = 5 ;
@@ -137,9 +137,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is test 2" ;
-		:maximum_name_length = 32 ;
 data:
 
  time_whole = 0.01, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/test2.dmp exodus-6.02/exodus/cbind/test/test2.dmp
--- exodus-6.02.orig/exodus/cbind/test/test2.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/test2.dmp	2016-09-09 10:59:34.379167514 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (0 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (0 currently)
 	num_dim = 3 ;
 	num_nodes = 26 ;
 	num_elem = 5 ;
@@ -119,9 +119,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  eb_status = 1, 1, 1, 1, 1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/test_clb.dmp exodus-6.02/exodus/cbind/test/test_clb.dmp
--- exodus-6.02.orig/exodus/cbind/test/test_clb.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/test_clb.dmp	2016-09-09 10:59:34.376167648 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (10 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (10 currently)
 	num_dim = 3 ;
 	num_nodes = 33 ;
 	num_elem = 7 ;
@@ -184,9 +184,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  time_whole = 0.01, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/testcp_dd.dmp exodus-6.02/exodus/cbind/test/testcp_dd.dmp
--- exodus-6.02.orig/exodus/cbind/test/testcp_dd.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/testcp_dd.dmp	2016-09-09 10:59:34.377167603 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (0 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (0 currently)
 	num_dim = 3 ;
 	num_nodes = 26 ;
 	num_elem = 5 ;
@@ -108,9 +108,9 @@
 // global attributes:
 		:floating_point_word_size = 8 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  eb_status = 1, 1, 1, 1, 1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/testcp_ds.dmp exodus-6.02/exodus/cbind/test/testcp_ds.dmp
--- exodus-6.02.orig/exodus/cbind/test/testcp_ds.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/testcp_ds.dmp	2016-09-09 10:59:34.377167603 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (0 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (0 currently)
 	num_dim = 3 ;
 	num_nodes = 26 ;
 	num_elem = 5 ;
@@ -108,9 +108,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  eb_status = 1, 1, 1, 1, 1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/testcp_ln.dmp exodus-6.02/exodus/cbind/test/testcp_ln.dmp
--- exodus-6.02.orig/exodus/cbind/test/testcp_ln.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/testcp_ln.dmp	2016-09-09 10:49:44.526607479 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (0 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (0 currently)
 	num_dim = 3 ;
 	num_nodes = 33 ;
 	num_elem = 7 ;
@@ -126,9 +126,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 0 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  eb_status = 1, 1, 1, 1, 1, 1, 1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/testcp_nl.dmp exodus-6.02/exodus/cbind/test/testcp_nl.dmp
--- exodus-6.02.orig/exodus/cbind/test/testcp_nl.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/testcp_nl.dmp	2016-09-09 10:55:56.609928955 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (0 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (0 currently)
 	num_dim = 3 ;
 	num_nodes = 33 ;
 	num_elem = 7 ;
@@ -128,9 +128,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  eb_status = 1, 1, 1, 1, 1, 1, 1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/testcp_sd.dmp exodus-6.02/exodus/cbind/test/testcp_sd.dmp
--- exodus-6.02.orig/exodus/cbind/test/testcp_sd.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/testcp_sd.dmp	2016-09-09 10:47:39.467213229 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (0 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (0 currently)
 	num_dim = 3 ;
 	num_nodes = 33 ;
 	num_elem = 7 ;
@@ -128,9 +128,9 @@
 // global attributes:
 		:floating_point_word_size = 8 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  eb_status = 1, 1, 1, 1, 1, 1, 1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/testcp_ss.dmp exodus-6.02/exodus/cbind/test/testcp_ss.dmp
--- exodus-6.02.orig/exodus/cbind/test/testcp_ss.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/testcp_ss.dmp	2016-09-09 10:46:42.709757365 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (0 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (0 currently)
 	num_dim = 3 ;
 	num_nodes = 33 ;
 	num_elem = 7 ;
@@ -128,9 +128,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  eb_status = 1, 1, 1, 1, 1, 1, 1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/testd.dmp exodus-6.02/exodus/cbind/test/testd.dmp
--- exodus-6.02.orig/exodus/cbind/test/testd.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/testd.dmp	2016-09-09 10:59:34.377167603 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (10 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (10 currently)
 	num_dim = 3 ;
 	num_nodes = 26 ;
 	num_elem = 5 ;
@@ -137,9 +137,9 @@
 // global attributes:
 		:floating_point_word_size = 8 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  time_whole = 0.01, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/test.dmp exodus-6.02/exodus/cbind/test/test.dmp
--- exodus-6.02.orig/exodus/cbind/test/test.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/test.dmp	2016-09-09 10:46:09.497246106 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (10 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (10 currently)
 	num_dim = 3 ;
 	num_nodes = 33 ;
 	num_elem = 7 ;
@@ -188,9 +188,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  time_whole = 0.01, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/testwt-long-name.dmp exodus-6.02/exodus/cbind/test/testwt-long-name.dmp
--- exodus-6.02.orig/exodus/cbind/test/testwt-long-name.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/testwt-long-name.dmp	2016-09-09 10:59:34.381167424 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (10 currently)
 	len_name = 128 ;
+	time_step = UNLIMITED ; // (10 currently)
 	num_dim = 3 ;
 	num_nodes = 33 ;
 	num_elem = 7 ;
@@ -188,9 +188,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 64 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 64 ;
 data:
 
  time_whole = 0.01, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/testwt-nfaced.dmp exodus-6.02/exodus/cbind/test/testwt-nfaced.dmp
--- exodus-6.02.orig/exodus/cbind/test/testwt-nfaced.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/testwt-nfaced.dmp	2016-09-09 10:59:34.380167469 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (0 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (0 currently)
 	num_dim = 3 ;
 	num_nodes = 14 ;
 	num_elem = 3 ;
@@ -47,9 +47,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  eb_status = 1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/testwt-nsided.dmp exodus-6.02/exodus/cbind/test/testwt-nsided.dmp
--- exodus-6.02.orig/exodus/cbind/test/testwt-nsided.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/testwt-nsided.dmp	2016-09-09 10:59:34.380167469 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (10 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (10 currently)
 	num_dim = 3 ;
 	num_nodes = 33 ;
 	num_elem = 7 ;
@@ -122,9 +122,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  time_whole = 0.01, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/testwt_ss.dmp exodus-6.02/exodus/cbind/test/testwt_ss.dmp
--- exodus-6.02.orig/exodus/cbind/test/testwt_ss.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/testwt_ss.dmp	2016-09-09 10:59:34.378167558 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (0 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (0 currently)
 	num_dim = 3 ;
 	num_nodes = 33 ;
 	num_elem = 8 ;
@@ -118,9 +118,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  eb_status = 1, 1, 1, 1, 1, 1, 1, 1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/testwt-zeroe.dmp exodus-6.02/exodus/cbind/test/testwt-zeroe.dmp
--- exodus-6.02.orig/exodus/cbind/test/testwt-zeroe.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/testwt-zeroe.dmp	2016-09-09 10:59:34.380167469 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (10 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (10 currently)
 	num_dim = 3 ;
 	num_nodes = 33 ;
 	num_node_sets = 2 ;
@@ -44,9 +44,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  time_whole = 0.01, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.1 ;
diff -Naur exodus-6.02.orig/exodus/cbind/test/testwt-zeron.dmp exodus-6.02/exodus/cbind/test/testwt-zeron.dmp
--- exodus-6.02.orig/exodus/cbind/test/testwt-zeron.dmp	2016-09-09 11:05:45.106549769 -0600
+++ exodus-6.02/exodus/cbind/test/testwt-zeron.dmp	2016-09-09 10:59:34.379167514 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (10 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (10 currently)
 	num_dim = 1 ;
 	num_qa_rec = 2 ;
 	num_info = 3 ;
@@ -20,9 +20,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  time_whole = 0.01, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.1 ;
diff -Naur exodus-6.02.orig/exodus/forbind/test/test1.dmp exodus-6.02/exodus/forbind/test/test1.dmp
--- exodus-6.02.orig/exodus/forbind/test/test1.dmp	2016-09-09 11:05:45.124548962 -0600
+++ exodus-6.02/exodus/forbind/test/test1.dmp	2016-09-09 11:03:20.246043098 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (10 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (10 currently)
 	num_dim = 3 ;
 	num_nodes = 28 ;
 	num_elem = 8 ;
@@ -174,9 +174,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is testwt1" ;
-		:maximum_name_length = 32 ;
 data:
 
  time_whole = 0.01, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.1 ;
diff -Naur exodus-6.02.orig/exodus/forbind/test/test2-1.dmp exodus-6.02/exodus/forbind/test/test2-1.dmp
--- exodus-6.02.orig/exodus/forbind/test/test2-1.dmp	2016-09-09 11:05:45.124548962 -0600
+++ exodus-6.02/exodus/forbind/test/test2-1.dmp	2016-09-09 11:03:20.246043098 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (10 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (10 currently)
 	num_dim = 3 ;
 	num_nodes = 26 ;
 	num_elem = 5 ;
@@ -137,9 +137,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is test 2" ;
-		:maximum_name_length = 32 ;
 data:
 
  time_whole = 0.01, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.1 ;
diff -Naur exodus-6.02.orig/exodus/forbind/test/test2-2.dmp exodus-6.02/exodus/forbind/test/test2-2.dmp
--- exodus-6.02.orig/exodus/forbind/test/test2-2.dmp	2016-09-09 11:05:45.124548962 -0600
+++ exodus-6.02/exodus/forbind/test/test2-2.dmp	2016-09-09 11:03:20.246043098 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (10 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (10 currently)
 	num_dim = 3 ;
 	num_nodes = 26 ;
 	num_elem = 5 ;
@@ -137,9 +137,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is test 2" ;
-		:maximum_name_length = 32 ;
 data:
 
  time_whole = 0.01, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.1 ;
diff -Naur exodus-6.02.orig/exodus/forbind/test/testcpln.dmp exodus-6.02/exodus/forbind/test/testcpln.dmp
--- exodus-6.02.orig/exodus/forbind/test/testcpln.dmp	2016-09-09 11:05:45.124548962 -0600
+++ exodus-6.02/exodus/forbind/test/testcpln.dmp	2016-09-09 11:03:20.243043233 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (0 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (0 currently)
 	num_dim = 3 ;
 	num_nodes = 26 ;
 	num_elem = 5 ;
@@ -106,9 +106,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 0 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  eb_status = 1, 1, 1, 1, 1 ;
diff -Naur exodus-6.02.orig/exodus/forbind/test/testcpnl.dmp exodus-6.02/exodus/forbind/test/testcpnl.dmp
--- exodus-6.02.orig/exodus/forbind/test/testcpnl.dmp	2016-09-09 11:05:45.124548962 -0600
+++ exodus-6.02/exodus/forbind/test/testcpnl.dmp	2016-09-09 11:03:20.243043233 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (0 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (0 currently)
 	num_dim = 3 ;
 	num_nodes = 26 ;
 	num_elem = 5 ;
@@ -108,9 +108,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  eb_status = 1, 1, 1, 1, 1 ;
diff -Naur exodus-6.02.orig/exodus/forbind/test/testcp_ss.dmp exodus-6.02/exodus/forbind/test/testcp_ss.dmp
--- exodus-6.02.orig/exodus/forbind/test/testcp_ss.dmp	2016-09-09 11:05:45.124548962 -0600
+++ exodus-6.02/exodus/forbind/test/testcp_ss.dmp	2016-09-09 11:03:20.243043233 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (0 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (0 currently)
 	num_dim = 3 ;
 	num_nodes = 26 ;
 	num_elem = 5 ;
@@ -108,9 +108,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  eb_status = 1, 1, 1, 1, 1 ;
diff -Naur exodus-6.02.orig/exodus/forbind/test/testd.dmp exodus-6.02/exodus/forbind/test/testd.dmp
--- exodus-6.02.orig/exodus/forbind/test/testd.dmp	2016-09-09 11:05:45.124548962 -0600
+++ exodus-6.02/exodus/forbind/test/testd.dmp	2016-09-09 11:03:20.246043098 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (10 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (10 currently)
 	num_dim = 2 ;
 	num_nodes = 8 ;
 	num_elem = 2 ;
@@ -97,9 +97,9 @@
 // global attributes:
 		:floating_point_word_size = 8 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  time_whole = 0.01, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.1 ;
diff -Naur exodus-6.02.orig/exodus/forbind/test/test.dmp exodus-6.02/exodus/forbind/test/test.dmp
--- exodus-6.02.orig/exodus/forbind/test/test.dmp	2016-09-09 11:05:45.124548962 -0600
+++ exodus-6.02/exodus/forbind/test/test.dmp	2016-09-09 11:03:20.243043233 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (10 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (10 currently)
 	num_dim = 3 ;
 	num_nodes = 26 ;
 	num_elem = 5 ;
@@ -137,9 +137,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  time_whole = 0.01, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.1 ;
diff -Naur exodus-6.02.orig/exodus/forbind/test/test-nsided.dmp exodus-6.02/exodus/forbind/test/test-nsided.dmp
--- exodus-6.02.orig/exodus/forbind/test/test-nsided.dmp	2016-09-09 11:05:45.124548962 -0600
+++ exodus-6.02/exodus/forbind/test/test-nsided.dmp	2016-09-09 11:03:20.246043098 -0600
@@ -3,8 +3,8 @@
 	len_string = 33 ;
 	len_line = 81 ;
 	four = 4 ;
-	time_step = UNLIMITED ; // (0 currently)
 	len_name = 33 ;
+	time_step = UNLIMITED ; // (0 currently)
 	num_dim = 3 ;
 	num_nodes = 33 ;
 	num_elem = 7 ;
@@ -40,9 +40,9 @@
 // global attributes:
 		:floating_point_word_size = 4 ;
 		:file_size = 1 ;
+		:maximum_name_length = 32 ;
 		:int64_status = 0 ;
 		:title = "This is a test" ;
-		:maximum_name_length = 32 ;
 data:
 
  eb_status = 1 ;
